### ISSUE TITLE
v12: api-frontend is not able to start due to RADIUS dictionnary missing

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#7114]
**Describe the bug**
Since merge of #7112, `api-frontend` is not able to start because FreeRADIUS is not installed anymore in `pfdebian` Docker images:

```
Aug 16 19:00:14 localhost api-frontend-docker-wrapper[113540]: dictionary: parse error in /usr/local/pf/lib/pf/util/combined_dictionary:2: open /usr/share/freeradius/dictionary: no such file or directory
Aug 16 19:00:14 localhost systemd[1]: Started PacketFence API frontend Service.
Aug 16 19:00:14 localhost api-frontend-docker-wrapper[113540]: panic: runtime error: invalid memory address or nil pointer dereference
```

**Comment by nqb** [comment]
Will certainly occured also for `pfacct` according to #7093.

@jrouzierinverse, do you have in mind other services that need access to `/usr/share/freeradius/dictionary` ?

**Comment by nqb** [comment]
ignore my previous comment, `pfacct` doesn't run inside container.

**Comment by nqb** [comment]
Fixes in 17cf2552441a5f5a7c5195fea8a157c57388bbb1, missing part of #7112

**Comment by nqb** [comment]
additionnal fix by @julsemaan in b8012ec8364d299ca24e4a6e2aeb5473c4b2a9da

**Comment by nqb** [comment]
RADIUS authentication source needs access to FreeRADIUS dictionnary. We need to find a better approach.
